### PR TITLE
Sort content labels lexicographically and allow setting of default product

### PR DESCRIPTION
**Pull request by matejak**
That way, the combo box contents are going to be consistent and e.g. RHEL8 will always come after RHEL7 and so on.

### CODE REVIEW

**Comment by jan-cerny** [comment]
Hi @matejak , what remains to be done here?

**Comment by matejak** [comment]
The problem is that this doesn't solve the primary UX problem, which is that we would like to offer RHEL8 option by default on RHEL8 systems, and RHEL7 on RHEL7 systems and so on. However, ordering may make sense by itself, in which case it can be merged.

**Comment by jan-cerny** [comment]
@matejak please rebase

**Comment by matejak** [comment]
The default item may now be controlled using a cmake argument `SCAP_WORKBENCH_PREFERRED_DATASTREAM_BASENAME`.

**Comment by jan-cerny** [comment]
@matejak Thanks for extending this. However, the default item setting works only sometimes. I suspect it conflicts with the `favorite` variable. If I set SCAP_WORKBENCH_PREFERRED_DATASTREAM_BASENAME to ssg-rhel7-ds.xml then RHEL7 is the default. But if I set it to ssg-ubuntu1804-ds.xml, Fedora is default, and Ubuntu 18.04 is only moved a little up, right after RHEL8.

I think that marking the default item as favourite would help us. At the same time I suggest dropping the favourite mechanism completely and only provide a lexicographically sorted list.

**Comment by matejak** [comment]
Good idea, the favourite concept shouldn't be defined in the sourcecode.
The new behavior allows specification of comma-separated list of datastream basenames, which allows compile-time composition analogy of the Favourites section, plus the ability to define order within the section.

File: CMakeLists.txt [hunk 1]
```diff
@@ -332,6 +332,8 @@ if (CPPCHECK_EXECUTABLE)
     )
 endif()
 
+set(SCAP_WORKBENCH_PREFERRED_DATASTREAM_BASENAMES "ssg-<productname1>-ds.xml" CACHE STRING "Comma-separated list of datastream that should be the first ones on the selection list in the initial dialogue.")
```
Comment (jan-cerny) [inline CMakeLists.txt]:
```suggestion
set(SCAP_WORKBENCH_PREFERRED_DATASTREAM_BASENAMES "ssg-<productname1>-ds.xml" CACHE STRING "Comma-separated list of datastreams that should be the first ones on the selection list in the initial dialogue.")
```

**Pull request merged**